templates/examples/docker_jenkins/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:14.04
+
+# Preparin dependencies
+RUN apt-get -y install wget
+RUN apt-get -y install software-properties-common
+RUN add-apt-repository ppa:webupd8team/java -y
+RUN apt-get update
+RUN echo oracle-java7-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
+RUN apt-get -y install oracle-java7-installer
```

Comment:
When i try to install openjdk it asking for more dependencies like jre, jre-headless and so on.